### DISCUSSION TITLE
Add support for updated Branch BlissRoms

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#22591] [Ideas]
* BlissRoms is now back to their old typhoon branch for july security patch 
* Havoc-OS released its thirteen branch, their twelve branch is always private from the beginning and useless till now

please merge https://github.com/SwapnilVicky/temporary/commit/13b2788f63fcb8f5615f7ca48f1588df63bdcd02

**Comment by Apon77** [comment]
may be they will change to typhon-qpr2 again. let us wait till then

**Reply by SwapnilVicky** [reply]
im an official maintainer for miatoll, they have confirmed they wont be moving to that branch again, after this typhoon they will move to a new branch for A14. If you want to wait then its okay for me :)

**Reply by Apon77** [reply]
mm, why did they change to qpr2 branch for only few months??

**Reply by SwapnilVicky** [reply]
because of QPR update by google, they dont want to mess the stable branch
